solidity/core/test/outbox.test.ts

```diff
@@ -123,16 +123,31 @@ describe('Outbox', async () => {
   });
 
   it('Checkpoints the latest root', async () => {
+    const message = ethers.utils.formatBytes32String('message');
+    const count = 10;
```

Comment:
Not a big deal but wondering if just making this `2` sufficiently illustrates the intent of the test and cuts down on the test's runtime